templates/patterns/featured_item/featured_item.ui_patterns.yml

```diff
@@ -0,0 +1,50 @@
+featured_item:
+  label: "Featured item"
+  description: "The featured item component uses a different visualisation to attract user attention. It's always the first item on the \"Latest section\"."
+  variants:
+    ~:
```

Comment:
Better to avoid empty variant names, I should actually make this not to pass validation in UI Patterns as it would break theme suggestions badly. Let's go for `default`.

Reply:
Done.